### PR TITLE
[FR] Update "Beatmap editor/Design" and "Beatmap editor/SB Load"

### PR DESCRIPTION
**Pull request by DoryanR**
Update of the translations of the following wiki pages:

- `Beatmap_Editor/Design`
- `Beatmap_Editor/SB_Load`

Also:

- Addition of the French images for the article `Design`

- [x] Waiting for a recheck from @Satsukel on some points seen together

### CODE REVIEW

File: wiki/Beatmap_Editor/Design/fr.md [hunk 1]
```diff
@@ -1,161 +1,158 @@
...
 
 #### Timeline
 
 | Nom | Description |
 | :-- | :-- |
-| Boutons `+` / `-` à gauche | Augmenter / Diminuer le zoom de la timeline |
-| Boutons fléchés `Haut` et `Bas` à gauche et en bas | Défilement vers le haut et vers le bas de la timeline de transformation (pour voir la timeline Déplacement/Couleur). |
-| Centre | Ligne de temps de transformation pour l'objet SB sélectionné. |
+| Boutons `+`/`-` à gauche | Augmenter/diminuer le zoom de la timeline |
+| Flèches `Haut`/`Bas` en bas à gauche | Faites défiler vers le haut/bas la timelie de transformation (pour voir la timeline des mouvements/couleurs). |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:36]:
```suggestion
| Flèches `Haut`/`Bas` en bas à gauche | Faites défiler vers le haut/bas la timeline de transformation (pour voir la timeline des mouvements/couleurs). |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 2]
```diff
@@ -1,161 +1,158 @@
...
+| Centre | Timeline de transformation pour l'objet SB sélectionné. |
 
-#### Contrôle de la Keyframe
+#### Contrôle des keyframe
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:39]:
```suggestion
#### Contrôle des keyframes
```

File: wiki/Beatmap_Editor/SB_Load/fr.md [hunk 3]
```diff
@@ -1,62 +1,59 @@
...
+Et c'est tout ! Votre arrière-plan sera remplacé par un mur noir solide, qui n'utilise pratiquement aucune puissance de traitement pour être dessiné. En particulier, comparé au fait de dessiner un arrière-plan plein écran, puis de dessiner un .png noir plein écran devant celui-ci, c'est une énorme amélioration !
 
 ### Évitez les espaces vides dans vos images
 
-Pour chaque png que vous utilisez dans votre storyboard,osu! doit dessiner le tout. Même les pixels transparents doivent être "dessinés", donc les images avec beaucoup d'espace vide imposent un stress inutile à l'ordinateur. C'est un problème facile à régler, en général :
+Pour chaque png que vous utilisez dans votre storyboard, osu! doit dessiner l'ensemble. Même les pixels transparents doivent être "dessinés", de sorte que les images comportant beaucoup d'espace vide imposent un stress inutile à l'ordinateur. C'est un problème facile à résoudre, en général :
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/SB_Load/fr.md:42]: Je chipotte un peu pour le "en" là j'avoue
```suggestion
Pour chaque png que vous utilisez dans votre storyboard, osu! doit en dessiner l'ensemble. Même les pixels transparents doivent être "dessinés", de sorte que les images comportant beaucoup d'espaces vides imposent un stress inutile à l'ordinateur. C'est un problème facile à résoudre, en général :
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Review by Satsukel — APPROVED** [review]

File: wiki/Beatmap_Editor/Design/fr.md [hunk 4]
```diff
@@ -1,161 +1,158 @@
...
+![Timeline du Storyboard](/wiki/shared/SE_STM.jpg "Timeline du Storyboard")
 
 #### Timeline
 
 | Nom | Description |
 | :-- | :-- |
-| Boutons `+` / `-` à gauche | Augmenter / Diminuer le zoom de la timeline |
-| Boutons fléchés `Haut` et `Bas` à gauche et en bas | Défilement vers le haut et vers le bas de la timeline de transformation (pour voir la timeline Déplacement/Couleur). |
-| Centre | Ligne de temps de transformation pour l'objet SB sélectionné. |
+| Boutons `+`/`-` à gauche | Augmenter/diminuer le zoom de la timeline |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:35]:
```suggestion
| Boutons `+`/`-` à gauche | Augmente/diminue le zoom de la timeline |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 5]
```diff
@@ -1,161 +1,158 @@
...
+| Flèches `Haut`/`Bas` en bas à gauche | Faites défiler vers le haut/bas la timeline de transformation (pour voir la timeline des mouvements/couleurs). |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:36]:
```suggestion
| Flèches `Haut`/`Bas` en bas à gauche | Fait défiler vers le haut/bas la timeline de transformation (pour voir la timeline des mouvements/couleurs). |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 6]
```diff
@@ -1,161 +1,158 @@
...
+#### Contrôle des keyframes
 
-Cela sera utilisé pour **ajouter des points d'ancrage (points de départ/fin).** Cela fonctionne à peu près de la même manière que les signets. Les boutons sont juste en dessous de la timeline.
+Il sera utilisé pour **ajouter des points d'ancrage (points de début/fin).** Il fonctionne à peu près de la même manière que les signets. Les boutons sont juste en dessous de la timeline.
 
 | Nom | Description |
 | :-- | :-- |
-| Boutons `+`/`-` | Ajouter/Supprimer des points d'ancrage pour la transformation sélectionnée. |
-| Boutons fléchés `Gauche` et `Droite` | Sauter en arrière/en avant vers le point d'ancrage le plus proche de la transformation sélectionnée. |
+| Boutons `+`/`-` | Ajouter/supprimer des points d'ancrage pour la transformation sélectionnée. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:45]:
```suggestion
| Boutons `+`/`-` | Ajoute/supprime des points d'ancrage pour la transformation sélectionnée. |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 7]
```diff
@@ -1,161 +1,158 @@
...
+| Flèches `Gauche`/`Droite` | Sauter en arrière/en avant vers le point d'ancrage le plus proche de la transformation sélectionnée. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:46]:
```suggestion
| Flèches `Gauche`/`Droite` | Saute en arrière/en avant vers le point d'ancrage le plus proche de la transformation sélectionnée. |
```

Comment (Satsukel) [reply wiki/Beatmap_Editor/Design/fr.md:46]: Ou sinon la variante à laquelle j'ai pensé aussi
```suggestion
| Flèches `Gauche`/`Droite` | Saute vers le point d'ancrage en arrière/en avant le plus proche de la transformation sélectionnée. |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 8]
```diff
@@ -1,161 +1,158 @@
...
 | :-- | :-- |
-| Déplacement | Traduction \[où aller (p. ex., se déplacer vers le haut)\]. |
-| Echelle | Agrandissement proportionnel (un carré reste un carré). |
-| Fondu | Fondu en entrée/sortie. (assombrir vs éclaircir) |
-| Faire pivoter | Faire pivoter un certain nombre de "Radians" (pas des degrés). |
-| Coloration | Forcer la coloration du sprite. Peut changer de couleur progressivement en utilisant une couleur différente comme point final. "Effet permanent". Explication de base. |
+| Translation | Translation \[où aller (par exemple, monter)\]. |
+| Homothétie | Agrandissement proportionnel (un carré reste un carré). |
+| Fondu | Fade in/out. (assombrir ou éclaircir) |
+| Rotation | Faire pivoter un certain nombre de "radian" (et non de degrés). |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:100]:
```suggestion
| Rotation | Fait pivoter un certain nombre de "radian" (et non de degrés). |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 9]
```diff
@@ -1,161 +1,158 @@
...
+![Extra commands](img/SE_AO-FR.jpg "Extra commands")
 
 | Commande | Utilisation |
 | :-- | :-- |
-| Tweening | Vous avez deux Keyframes (ie Start/End), voulez-vous animer entre elles ? Si oui, allumez Tweening. Sinon, désactivez-le. |
-| Entrée/sortie de l'effacement | Démarrage lent / rapide puis rapide / lent près du point final. Utile pour le fondu enchaîné. |
-| Origine | Point d'origine. Coin "Haut-Gauche" ou "Centre" de l'image |
-| Diff. Spécifique | Sauvegardez tous les SB dans le fichier `.osu` de cette difficulté au lieu du fichier `.osb`. (Voir note ci-dessous) |
+| Interpolation d'animation | Vous avez deux images clés (c'est-à-dire au début et à la fin), voulez-vous animer entre elles ? Si oui, activez l'interpolation d'animation. Sinon, désactivez-la. |
+| Style d'animation | Commencez lentement/rapidement puis vite/lent vers le point final. Utile avec le Fade In. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:117]:
```suggestion
| Style d'animation | Commence lentement/rapidement puis vite/lent vers le point final. Utile avec le Fade In. |
```

File: wiki/Beatmap_Editor/Design/fr.md [hunk 10]
```diff
@@ -1,161 +1,158 @@
...
+| Origine | Point d'origine. Coin "en haut à gauche" ou "centre" de l'image. |
+| Spécifique à cette diff | Sauvegardez tous les SB dans le fichier `.osu` de cette difficulté au lieu du fichier `.osb`. |
```
Comment (Satsukel) [inline wiki/Beatmap_Editor/Design/fr.md:119]:
```suggestion
| Spécifique à cette diff | Sauvegarde tous les SB dans le fichier `.osu` de cette difficulté au lieu du fichier `.osb`. |
```

**Review by Satsukel — CHANGES_REQUESTED** [review]

**Review by Satsukel — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**